llama.cpp

```diff
@@ -2323,6 +2320,37 @@ static struct ggml_cgraph * llm_build_llama(
                 ggml_build_forward_expand(gf, ggml_cpy(ctx0, Vcur, v));
             }
 
+#ifdef LLAMA_SKIP_UNUSED_LOGITS
+            if (il == n_layer - 1 && !lctx.logits_all)
+            {
```

Comment:
This does not follow the [coding guidelines in the README](https://github.com/ggerganov/llama.cpp#coding-guidelines).

Reply:
Thanks, fixed.